### PR TITLE
PMM-1357 adding tooltips to MySQL Replication dashboard

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: dashboards/MySQL_Replication.json [hunk 1]
```diff
@@ -720,6 +810,7 @@
             "dashLength": 10,
             "dashes": false,
             "datasource": "Prometheus",
+            "description": "Shows the amount of data written to the binlog files during the last 24 hours and grouped into hourly buckets. This graph can give you an idea of the amount of changes to your data.",
```
Comment (adivinho) [inline dashboards/MySQL_Replication.json]: IMHO the notice about 24 hours has to be removed.
It's true only if quick range - "Last 24 hours" is used.
screenshot_20181114_100203

File: dashboards/MySQL_Replication.json [hunk 2]
```diff
@@ -918,6 +1023,7 @@
             "dashLength": 10,
             "dashes": false,
             "datasource": "Prometheus",
+            "description": "Shows the number of binlog files created hourly during the last 24 hours. The number depends on the rotation policy adopted using the configuration variables **max_binlog_size** and **expire_logs_days**.",
```
Comment (adivinho) [inline dashboards/MySQL_Replication.json]:
```suggestion
            "description": "Shows the number of binlog files created hourly. The number depends on the rotation policy adopted using the configuration variables **max_binlog_size** and **expire_logs_days**.",
```

File: dashboards/MySQL_Replication.json [hunk 3]
```diff
@@ -1116,6 +1230,7 @@
             "dashLength": 10,
             "dashes": false,
             "datasource": "Prometheus",
+            "description": "Shows the amount of data written hourly to the relay log files during the last 24 hours.",
```
Comment (adivinho) [inline dashboards/MySQL_Replication.json]:
```suggestion
            "description": "Shows the amount of data written hourly to the relay log files.",
```

**Comment by rnovikovP** [comment]
@fiowro  pls check  ID of this  PR. looks like it's  from another  Jira ID

**Comment by rnovikovP** [comment]
@michaelcoburn  @adivinho  pls review this

**Pull request merged**